css/properties/-webkit-mask-repeat-x.json

```diff
@@ -0,0 +1,59 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-mask-repeat-x": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-repeat-x",
+          "support": {
+            "chrome": {
+              "version_added": false,
```

Comment:
Should this be true and partial_implementation true as well?

Reply:
x and y are supported in both Android and Webview.

Reply:
Ah I was under the impression that "While the property itself is recognized, it doesn't accept any values." would be false but that's addressed here: #845

Maybe we should mention this in the docs?

@jpmedley supported fully or just recognized? And do you know if that's true of desktop Chrome as well?